packages/commands/src/account-managers.mjs

```diff
@@ -1,27 +1,15 @@
 import chalk from 'chalk'
 import { program } from 'commander'
 
-import { initializeAudiusLibs, initializeAudiusSdk } from './utils.mjs'
+import { initializeAudiusSdk } from './utils.mjs'
 
 program
   .command('add-manager')
   .description('Create a pending manager request for a user')
   .argument('[handle]', 'The handle of the manager')
   .option('-f, --from <from>', 'The account that will be managed')
   .action(async (handle, { from }) => {
-    const audiusLibs = await initializeAudiusLibs(from)
-    // extract privkey and pubkey from hedgehog
-    // only works with accounts created via audius-cmd
-    const wallet = audiusLibs?.hedgehog?.getWallet()
-    const privKey = wallet?.getPrivateKeyString()
-    const pubKey = wallet?.getAddressString()
-
-    // init sdk with priv and pub keys as api keys and secret
-    // this enables writes via sdk
-    const audiusSdk = await initializeAudiusSdk({
-      apiKey: pubKey,
-      apiSecret: privKey
-    })
+    const audiusSdk = await initializeAudiusSdk({ handle: from })
```

Comment:
why don't we need to initialize with the api stuff anymore?

Reply:
It's the difference between how `AppAuth` (meant to be used by third-party apps) and `UserAuth`(meant to be used by a user) work.

We were kind of abusing the `AppAuth` concept here by passing in the private key from hedgehog and the corresponding public key. `UserAuth` has Hedgehog embedded inside it and doesn't need you to pass public/private keys into it (it derives them from the entropy, which for audius-cmd is in localStorage).

This is more directly equivalent to how we use SDK in client.